app/controllers/world_location_news_controller.rb

```diff
@@ -30,4 +38,23 @@ def index
   def load_world_location
     @world_location = WorldLocation.with_translations(I18n.locale).find(params[:world_location_id])
   end
+
+  def fetch_documents(additional_options = {})
```

Comment:
After i've merged my pr is the plan to refactor this code to use the SearchRummagerService?

Reply:
Yep that's right 😸